scipy/sparse/csgraph/_flow.pyx

```diff
@@ -45,6 +47,17 @@ def maximum_flow(csgraph, source, sink):
         The source vertex from which the flow flows.
     sink : int
         The sink vertex to which the flow flows.
+    
+    .. versionadded:: 1.8.0
+
+    method: str
+        The method/algorithm to be used for computing the maximum flow.
+        Following methods are supported,
+
+            * 'edmonds_karp': Edmonds Karp algorithm in [1].
+            * 'dinic': Dinic's algorithm in [4].
+
+        Optional, by default, 'edmonds_karp'.
 
```

Comment:
Some formating for the links and the version. You can check the result of the doc build `ci/circleci: build_docs artifact `
```suggestion
        The sink vertex to which the flow flows.
    method: {‘edmonds_karp’, ‘dinic’}, optional
        The method/algorithm to be used for computing the maximum flow.
        Following methods are supported,

            * 'edmonds_karp': Edmonds Karp algorithm in [1]_.
            * 'dinic': Dinic's algorithm in [4]_.

        Default is 'edmonds_karp'.

        .. versionadded:: 1.8.0

```